app/controllers/content_pages_controller.rb

```diff
@@ -18,7 +18,7 @@ def show
     elsif module_item.assessment_results?
       redirect_to training_module_assessment_result_path(training_module_name, module_item)
     else
-      render content_page_partial(module_item)
+      render_content_page_partial
     end
```

Comment:
Just a bit of pedantry here - this is actually rendering a content page template, not a partial, I think just `render_content_page` is sufficient.